source/glimmer-components/index.md

```diff
@@ -0,0 +1,37 @@
+## HTML elements as an abstraction for text documents
+
```

Comment:
I think it makes sense for there to be a "Philosophy of Ember Components" guide that talks about the programming model at a high level in a non-tutorial style. Is that what you have in mind?